compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -17762,15 +17762,29 @@ TR::Node *fmaxminSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier *
          fmin = fmax = secondChild->getFloat();
       else
          {
-         if (firstChild->getFloat() <= secondChild->getFloat())
+         float first = firstChild->getFloat();
+         float second = secondChild->getFloat();
+         int nZeroBits = 0x80000000;
+         if (first <= second)
             {
-            fmin = firstChild->getFloat();
-            fmax = secondChild->getFloat();
+            if (first == second && first == 0)
+               {
+               int firstBits = *(int *)(&first);
+               int secondBits = *(int *)(&second);
+               int sign = firstBits & nZeroBits;
```

Comment:
Do we need to. zero out remaining bits here ?

Reply:
shouldn't need to right? the bitwise And with `nZeroBits` should zero all but the most significant bit

Reply:
I meant, if the float is negative 0, can that be represented by some other bit pattern other than `0x80000000` ?